LegendaryExplorer/LegendaryExplorerCore/Save/HeadMorph.cs

```diff
@@ -163,10 +164,10 @@ private static KeyValuePair<string, LinearColor> GetKeyedVector(string[] keyValS
             var vectStr = keyValSplit[1].Trim().Trim('(', ')', ',').Split(',');
             return new KeyValuePair<string, LinearColor>(fn, new LinearColor()
             {
-                R = float.Parse(vectStr[0]),
-                G = float.Parse(vectStr[1]),
-                B = float.Parse(vectStr[2]),
-                A = float.Parse(vectStr[3]),
+                R = float.Parse(vectStr[0], CultureInfo.InvariantCulture),
+                G = float.Parse(vectStr[1], CultureInfo.InvariantCulture),
+                B = float.Parse(vectStr[2], CultureInfo.InvariantCulture),
+                A = float.Parse(vectStr[3], CultureInfo.InvariantCulture),
```

Comment:
We should look across the LEX and LEC codebases for float.Parse and float.TryParse to look for issues like this.